datatypes/src/main/java/org/hyperledger/besu/datatypes/Address.java

```diff
@@ -71,7 +71,8 @@ public class Address extends DelegatingBytes implements org.hyperledger.besu.plu
   /** The constant BLS12_MAP_FP2_TO_G2. */
   public static final Address BLS12_MAP_FP2_TO_G2 = Address.precompiled(0x12);
 
-  /** The constant ZERO. */
+  public static final Address KZG_POINT_EVAL = Address.precompiled(0x14);
```

Comment:
Is this address fixed already?  Has BLS been "locked in" to those addresses?

Reply:
As far as i know it is, the specs have been stable on it.